huaweicloud/resource_huaweicloud_obs_bucket_test.go

```diff
@@ -380,6 +380,16 @@ resource "huaweicloud_obs_bucket" "bucket" {
 `, randInt, randInt)
 }
 
+func testAccObsBucketConfigWithQuota(randInt int) string {
```

Comment:
Seems the acceptance test is uncompleted. 
[Terraform acceptance tests](https://www.terraform.io/docs/extend/testing/acceptance-tests/testcase.html) are declared with the naming pattern TestAccXxx.

Reply:
@ShiChangkuo done.